h/search/index.py

```diff
@@ -34,18 +33,14 @@ def __init__(self, session, es_client, request, target_index=None, op_type="inde
         else:
             self._target_index = target_index
 
-    def index(
-        self, annotation_ids=None, windowsize=PG_WINDOW_SIZE, chunk_size=ES_CHUNK_SIZE
-    ):
+    def index(self, annotation_ids=None, windowsize=PG_WINDOW_SIZE):
```

Comment:
I think it's good to remove this from the method call, but I think it's a shame to remove it over all. Even if it documents the value, it gives us an obvious handle to tweak, rather than meaning you have to find the docs or guess.